sdk/core/azure-core/src/main/java/com/azure/core/http/rest/RestProxy.java

```diff
@@ -545,8 +547,9 @@ private static void endTracingSpan(Signal<HttpDecodedResponse> signal) {
         // Get the context that was added to the mono, this will contain the information needed to end the span.
         reactor.util.context.Context context = signal.getContext();
         Optional<Context> tracingContext = context.getOrEmpty("TRACING_CONTEXT");
+        Optional<Boolean> disableTracing = context.getOrEmpty(Tracer.DISABLE_TRACING_KEY);
```

Comment:
Do we want to do the same thing as above where we use a `orElse`?

Reply:
Done.